### PR TITLE
docs command added

### PR DESCRIPTION
**Pull request by SenjeyB**
Added docs command, which does should print 'Hello World'.

### CODE REVIEW

**Comment by SenjeyB** [comment]
@yegor256 Can you take a look, please?

**Comment by yegor256** [comment]
@SenjeyB please, make sure all CI jobs pass before manual review

File: src/commands/docs.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+/**
+ * Command to create docs from .EO sources.
+ * @param {Hash} opts - All options
+ */
+module.exports = function(opts) {
+  console.info('Hello World!');
```
Comment (yegor256) [inline src/commands/docs.js:30]: @SenjeyB maybe it's better to print something like `"This command is not yet implemented, sorry"`

**Comment by SenjeyB** [comment]
> @SenjeyB looks good now, but how about a test? Adding new code not covered by tests is a bad idea in general.

Tests added

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@SenjeyB thanks!

**Pull request merged**